modules/shortcodes/slideshare.php

```diff
@@ -111,5 +111,10 @@ function slideshare_shortcode( $atts ) {
 
 	$player .= ' allowfullscreen webkitallowfullscreen mozallowfullscreen></iframe>';
 
-	return $player;
+	/**
+	 * Filter the returned shortcode.
+	 *
+	 * @param string $player The iframe to return.
+	 */
+	return apply_filters( 'slideshare_shortcode', $player );
```

Comment:
There only seems to be one place wpcom-side where this filter is used, so it's probably changeable in a subsequent PR -- but for simplicity in sync sake, I'd prefer to merge it as is, and then do that one.

I'd be delighted to add `$attr` as well in a subsequent PR.